content/guides/guides/cross-origin-testing.md

```diff
@@ -17,6 +17,8 @@ a navigation occurs that does not meet the same superdomain rule, the
 [`cy.origin()`](/api/commands/origin) command must be used to execute Cypress
 commands inside the newly navigated origin.
 
+::include{file=partials/document-domain-workaround.md}
```

Comment:
I think this plug should be at the end of the doc as an FAQ not the beginning since it should be an edge case. I also think additional context of why this might be useful could be helpful.

Reply:
It does feel awkward. Going to see if I can get something to flow as an FAQ at the end